src/mono/mono/metadata/custom-attrs-types.h

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
This could be added to custom-attrs-internals.h instead.

Reply:
Moved to `custom-attrs-internals.h` as suggested